cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/TranslationManager.kt

```diff
@@ -276,7 +270,7 @@ private constructor(
                 Thread.currentThread().interrupt()
             } catch (e: ExecutionException) {
                 log.error("Error parsing ${futureToFile[future]}", e)
-                Thread.currentThread().interrupt()
+                // Thread.currentThread().interrupt()
```

Comment:
Partly. I was not 100 % sure why it is there; and I actually had problems that crashed the execution of the CPG with an interruption that seems to be thrown by this. I know that you should re-throw the interrupt in case of an `InterruptException` (the lines above), but I am not so sure about an `ExectionException`.